IFC4x3/PropertyEnumerations/l/PEnum_LateralOffsetDirection.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="PEnum_LateralOffsetDirection" UniqueId="05b8a5c7-9f76-4895-bc0d-f461d3121121">
+	<Documentation>Gives the options for positive lateral offset direction. The values are _LRM_LEFT_ and _LRM_RIGHT_. This lateral offset direction is as viewed from above the linear element facing in the direction of increasing measure.
```

Comment:
Same as up and down. Just knowing how the linear element is increasing its length measure does not help without the initial context.